appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -1789,10 +1789,12 @@
           360))
 
 (define (sin-degrees degrees)
-  (sin (degrees->radians-internal degrees)))
+  (if (or (= degrees 180) (= degrees 360)) 0
```

Comment:
I think we want to solve this more generally by detecting when `(= 0 (mod degrees 180))` rather than just testing for the specific examples given in the issue.